### PR TITLE
Fixed can't access property getUnits, sourceProj is null

### LINKED ISSUE #19266: Uncaught TypeError: can't access property "getUnits", sourceProj is null
### Describe the bug

`Uncaught TypeError: can't access property "getUnits", sourceProj is null` when entering an invalid SRID number.

### To Reproduce

1. Go to https://demo.phpmyadmin.net/master/public/index.php?route=/sql&db=opengis&table=world_borders&pos=0&server=1
2. Click on `Edit`
3. Click on `Edit/Insert`
4. Click on `Use OpenStreetMaps as Base Layer`
5. Enter a random SRID
6. See error

### Expected behavior

No error should happen.

### Screenshots

https://github.com/user-attachments/assets/6d83e32a-545a-4ff9-abda-2780e231391d

### Server configuration

- phpMyAdmin version: 6.0.0-dev

### Additional context

```JSON
{
    "pma_version": "6.0.0-dev+20240817.dcc741cc3d",
    "browser_name": "FIREFOX",
    "browser_version": "131.0",
    "user_os": "Win",
    "server_software": "nginx/1.27.1",
    "user_agent_string": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:131.0) Gecko/20100101 Firefox/131.0",
    "locale": "en",
    "configuration_storage": "enabled",
    "php_version": "8.4.0beta3",
    "script_name": "index.php",
    "exception_type": "js",
    "exception": {
        "mode": "stack",
        "name": "TypeError",
        "message": "can't access property \"getUnits\", sourceProj is null",
        "stack": [
            {
                "func": "transform",
                "line": "7019",
                "column": "7",
                "context": [
                    "   */",
                    "  transform(source, destination) {",
                    "    /** @type {import(\"../proj/Projection.js\").default} */",
                    "    const sourceProj = proj_get(source);",
                    "    const transformFn =",
                    "      sourceProj.getUnits() == 'tile-pixels'",
                    "        ? function (inCoordinates, outCoordinates, stride) {",
                    "            const pixelExtent = sourceProj.getExtent();",
                    "            const projectedExtent = sourceProj.getWorldExtent();",
                    "            const scale = getHeight(projectedExtent) / getHeight(pixelExtent);",
                    "            compose("
                ],
                "uri": "js/vendor/openlayers/OpenLayers.js?",
                "scriptname": "js/vendor/openlayers/OpenLayers.js"
            },
            {
                "func": "getFeaturesFromOpenLayersData",
                "line": "359",
                "column": "31",
                "context": [
                    "      style.stroke = new window.ol.style.Stroke(geometry.style.stroke);",
                    "    } else {",
                    "      throw new Error();",
                    "    }",
                    "    if (geometry.geometry.srid !== 3857) {",
                    "      olGeometry = olGeometry.transform('EPSG:' + (geometry.geometry.srid !//...",
                    "    }",
                    "    if (geometry.style.text) {",
                    "      style.text = new window.ol.style.Text(geometry.style.text);",
                    "    }",
                    "    const feature = new window.ol.Feature(olGeometry);"
                ],
                "uri": "js/dist/table/gis_visualization.js?",
                "scriptname": "js/dist/table/gis_visualization.js"
            },
            {
                "func": "drawOpenLayers",
                "line": "275",
                "column": "17",
                "context": [
                    "    if (typeof window.ol === 'undefined') {",
                    "      return undefined;",
                    "    }",
                    "    jquery__WEBPACK_IMPORTED_MODULE_0___default()('head').append('<link rel//...",
                    "    const vectorSource = new window.ol.source.Vector({",
                    "      features: getFeaturesFromOpenLayersData(this.data)",
                    "    });",
                    "    const map = new window.ol.Map({",
                    "      target: this.target,",
                    "      layers: [new window.ol.layer.Tile({",
                    "        source: new window.ol.source.OSM()"
                ],
                "uri": "js/dist/table/gis_visualization.js?",
                "scriptname": "js/dist/table/gis_visualization.js"
            },
            {
                "func": "show",
                "line": "306",
                "column": "25",
                "context": [
                    "  show() {",
                    "    super.show();",
                    "    if (this.olMap) {",
                    "      this.olMap.updateSize();",
                    "    } else {",
                    "      this.olMap = this.drawOpenLayers();",
                    "    }",
                    "  }",
                    "  dispose() {",
                    "    if (this.olMap) {",
                    "      // Removes ol.Map's resize listener from window"
                ],
                "uri": "js/dist/table/gis_visualization.js?",
                "scriptname": "js/dist/table/gis_visualization.js"
            },
            {
                "func": "selectVisualization",
                "line": "407",
                "column": "12",
                "context": [
                    "      if (!this.svgVis) {",
                    "        this.svgVis = new SvgVisualization(jquery__WEBPACK_IMPORTED_MODULE_//...",
                    "      }",
                    "      newVis = this.svgVis;",
                    "    }",
                    "    newVis.show();",
                    "  }",
                    "  /**",
                    "   * Cleanup events when no longer needed",
                    "   */",
                    "  dispose() {"
                ],
                "uri": "js/dist/table/gis_visualization.js?",
                "scriptname": "js/dist/table/gis_visualization.js"
            },
            {
                "func": "GisVisualizationController",
                "line": "381",
                "column": "10",
                "context": [
                    "    jquery__WEBPACK_IMPORTED_MODULE_0___default()(document).on('click', '#u//...",
                    "    if (typeof window.ol === 'undefined') {",
                    "      jquery__WEBPACK_IMPORTED_MODULE_0___default()('#useOsmAsBaseLayerSwit//...",
                    "      jquery__WEBPACK_IMPORTED_MODULE_0___default()('#useOsmAsBaseLayerSwit//...",
                    "    }",
                    "    this.selectVisualization();",
                    "  }",
                    "  onChoiceChange() {",
                    "    this.selectVisualization();",
                    "  }",
                    "  /**"
                ],
                "uri": "js/dist/table/gis_visualization.js?",
                "scriptname": "js/dist/table/gis_visualization.js"
            },
            {
                "func": "initGISEditorVisualization",
                "line": "34",
                "column": "29",
                "context": [
                    "}",
                    "/**",
                    " * Initialize the visualization in the GIS data editor.",
                    " */",
                    "function initGISEditorVisualization(olData) {",
                    "  visualizationController = new window.GisVisualizationController(olData);",
                    "}",
                    "function withIndex(prefix) {",
                    "  let result = prefix;",
                    "  for (let i = 0; i < (arguments.length <= 1 ? 0 : arguments.length - 1); ++i) {",
                    "    result += '[' + (i + 1 < 1 || arguments.length <= i + 1 ? undefined : a//..."
                ],
                "uri": "js/dist/gis_data_editor.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/dist/gis_data_editor.js"
            },
            {
                "func": "onCoordinateEdit",
                "line": "244",
                "column": "29",
                "context": [
                    "    return;",
                    "  }",
                    "  disposeGISEditorVisualization();",
                    "  jquery__WEBPACK_IMPORTED_MODULE_0___default()('#visualization-placeholder//...",
                    "  jquery__WEBPACK_IMPORTED_MODULE_0___default()('#gis_data_textarea').val(d//...",
                    "  initGISEditorVisualization(data.openLayersData);",
                    "}",
                    "/**",
                    " * Handles adding data points",
                    " */",
                    "function addPoint() {"
                ],
                "uri": "js/dist/gis_data_editor.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/dist/gis_data_editor.js"
            },
            {
                "func": "c",
                "line": "2",
                "column": "25304",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "fireWith",
                "line": "2",
                "column": "26053",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "l",
                "line": "2",
                "column": "77782",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "send/o/<",
                "line": "2",
                "column": "80265",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "EventHandlerNonNull*send",
                "line": "2",
                "column": "80451",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "ajax",
                "line": "2",
                "column": "76167",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "?",
                "line": "2",
                "column": "4771",
                "context": [
                    "/*! jQuery Migrate v3.5.0 | (c) OpenJS Foundation and other contributors | //...",
                    "\"undefined\"==typeof jQuery.migrateMute&&(jQuery.migrateMute=!0),function(t)//...",
                    "//# sourceMappingURL=jquery-migrate.min.map"
                ],
                "uri": "js/vendor/jquery/jquery-migrate.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery-migrate.min.js"
            },
            {
                "func": "a/e[t]",
                "line": "2",
                "column": "1582",
                "context": [
                    "/*! jQuery Migrate v3.5.0 | (c) OpenJS Foundation and other contributors | //...",
                    "\"undefined\"==typeof jQuery.migrateMute&&(jQuery.migrateMute=!0),function(t)//...",
                    "//# sourceMappingURL=jquery-migrate.min.map"
                ],
                "uri": "js/vendor/jquery/jquery-migrate.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery-migrate.min.js"
            },
            {
                "func": "ce[i]",
                "line": "2",
                "column": "78232",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "updateResult",
                "line": "324",
                "column": "49",
                "context": [
                    " */",
                    "function updateResult() {",
                    "  const $form = jquery__WEBPACK_IMPORTED_MODULE_0___default()('form#gis_dat//...",
                    "  const argsep = _modules_common_ts__WEBPACK_IMPORTED_MODULE_2__.CommonPara//...",
                    "  const data = $form.serialize() + argsep + 'generate=true' + argsep + 'aja//...",
                    "  jquery__WEBPACK_IMPORTED_MODULE_0___default().post('index.php?route=/gis-//...",
                    "}",
                    "/**",
                    " * Unbind all event handlers before tearing down a page",
                    " */",
                    "_modules_ajax_ts__WEBPACK_IMPORTED_MODULE_1__.AJAX.registerTeardown('gis_da//..."
                ],
                "uri": "js/dist/gis_data_editor.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/dist/gis_data_editor.js"
            },
            {
                "func": "dispatch",
                "line": "2",
                "column": "40035",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "add/v.handle",
                "line": "2",
                "column": "38006",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "EventListener.handleEvent*add",
                "line": "2",
                "column": "38477",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "?",
                "line": "2",
                "column": "10057",
                "context": [
                    "/*! jQuery Migrate v3.5.0 | (c) OpenJS Foundation and other contributors | //...",
                    "\"undefined\"==typeof jQuery.migrateMute&&(jQuery.migrateMute=!0),function(t)//...",
                    "//# sourceMappingURL=jquery-migrate.min.map"
                ],
                "uri": "js/vendor/jquery/jquery-migrate.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery-migrate.min.js"
            },
            {
                "func": "a/e[t]",
                "line": "2",
                "column": "1582",
                "context": [
                    "/*! jQuery Migrate v3.5.0 | (c) OpenJS Foundation and other contributors | //...",
                    "\"undefined\"==typeof jQuery.migrateMute&&(jQuery.migrateMute=!0),function(t)//...",
                    "//# sourceMappingURL=jquery-migrate.min.map"
                ],
                "uri": "js/vendor/jquery/jquery-migrate.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery-migrate.min.js"
            },
            {
                "func": "Le/<",
                "line": "2",
                "column": "37118",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "each",
                "line": "2",
                "column": "3129",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "each",
                "line": "2",
                "column": "1594",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "Le",
                "line": "2",
                "column": "37093",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "on",
                "line": "2",
                "column": "44478",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "60/onloadNavigation/<",
                "line": "449",
                "column": "61",
                "context": [
                    "        $iconThrobberSrc.hide();",
                    "      }",
                    "      // Reload the navigation",
                    "      _navigation_ts__WEBPACK_IMPORTED_MODULE_2__.Navigation.reload(hideNav);",
                    "    });",
                    "    jquery__WEBPACK_IMPORTED_MODULE_0___default()(document).on('change', '#//...",
                    "      if (!jquery__WEBPACK_IMPORTED_MODULE_0___default()(this).val()) {",
                    "        _navigation_ts__WEBPACK_IMPORTED_MODULE_2__.Navigation.update(_comm//...",
                    "        _navigation_ts__WEBPACK_IMPORTED_MODULE_2__.Navigation.reload();",
                    "      }",
                    "      jquery__WEBPACK_IMPORTED_MODULE_0___default()(this).closest('form').t//..."
                ],
                "uri": "js/dist/main.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/dist/main.js"
            },
            {
                "func": "e",
                "line": "2",
                "column": "27028",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "Deferred/then/l/</t<",
                "line": "2",
                "column": "27330",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "setTimeout handler*Deferred/then/l/<",
                "line": "2",
                "column": "27594",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "c",
                "line": "2",
                "column": "25304",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "fireWith",
                "line": "2",
                "column": "26053",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "fire",
                "line": "2",
                "column": "26089",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "c",
                "line": "2",
                "column": "25304",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "fireWith",
                "line": "2",
                "column": "26053",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "ready",
                "line": "2",
                "column": "29112",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "P",
                "line": "2",
                "column": "28884",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "EventListener.handleEvent*",
                "line": "2",
                "column": "29264",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "?",
                "line": "2",
                "column": "220",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            },
            {
                "func": "?",
                "line": "2",
                "column": "225",
                "context": [
                    "/*! jQuery v3.7.1 | (c) OpenJS Foundation and other contributors | jquery.o//...",
                    "!function(e,t){\"use strict\";\"object\"==typeof module&&\"object\"==typeof modul//...",
                    ""
                ],
                "uri": "js/vendor/jquery/jquery.min.js?v=6.0.0-dev%2B20240817.dcc741cc3d",
                "scriptname": "js/vendor/jquery/jquery.min.js"
            }
        ],
        "uri": "index.php?route=%2Ftable%2Fchange&where_clause=%60uk%60.%60Country%60+%3D+%27Scotland%27&where_clause_signature=25dd6bd34a5a5368049b9972dd977001d0a1126ab88c3459bb5e0e2534f99abb&clause_is_unique=1&sql_query=SELECT+%2A+FROM+%60uk%60&sql_signature=4bd04ea939a468d3143c39c8b27417fce88d6315e7995d0741aeb75224844869&goto=index.php%3Froute%3D%2Fsql%26server%3D2&default_action=update"
    }
}
```

### PR DESCRIPTION
**Pull request by SachinBahukhandi**
### Description

Fixes the issue coming up upon entering an invalid SRID.
Now the function would execute only if the `sourceProj` is a valid object.

Fixes: https://github.com/phpmyadmin/phpmyadmin/issues/19266

### CODE REVIEW

File: resources/js/src/table/gis_visualization.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import $ from 'jquery';
 import { AJAX } from '../modules/ajax.ts';
 import { escapeHtml } from '../modules/functions/escape.ts';
+import { get as getProjection } from 'ol/proj.js';
```
Comment (MauricioFauth) [inline resources/js/src/table/gis_visualization.ts:4]: Could you move that to the `resources/js/src/ol.mjs` file? We are not directly importing third-party dependencies yet.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

File: resources/js/src/table/gis_visualization.ts (hunk 1 above)
Comment (SachinBahukhandi) [reply resources/js/src/table/gis_visualization.ts:4]: But won't we have to reimport the `ol.mjs` file inside the above file?

Comment (MoonE) [reply resources/js/src/table/gis_visualization.ts:4]: `ol` is availabe as a property of `window`
https://github.com/phpmyadmin/phpmyadmin/blob/71fac7eb4c992c28e619327341170e7f157e0b3e/resources/js/src/table/gis_visualization.ts#L437-L455

`OpenLayers.js` is  added in a script tag. 
https://github.com/phpmyadmin/phpmyadmin/blob/71fac7eb4c992c28e619327341170e7f157e0b3e/src/Controllers/Table/GisVisualizationController.php#L128

which is transpiled from `ol.mjs`
https://github.com/phpmyadmin/phpmyadmin/blob/71fac7eb4c992c28e619327341170e7f157e0b3e/webpack.config.cjs#L158-L183

Comment (SachinBahukhandi) [reply resources/js/src/table/gis_visualization.ts]: @MoonE thanks for the clarity. It was really helpful.

Comment (SachinBahukhandi) [reply resources/js/src/table/gis_visualization.ts]: @MauricioFauth please re-review.

**Review by MauricioFauth — APPROVED** [review]

**Comment by MauricioFauth** [comment]
Merged! Thank you!

**Pull request merged**